certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelope-descriptors/{envelope-descriptor-id}/envelopes": {
+      "post": {
+        "operationId": "PostEnvelopeDescriptorsEnvelopeDescriptorIdEnvelopes",
+        "summary": "Create envelope - descriptor",
+        "description": "Creates an envelope based on a specific descriptor.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeDescriptorId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/EnvelopeInput"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "uid": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/documents": {
+      "get": {
+        "operationId": "GetEnvelopesEnvelopeIdDocuments",
+        "summary": "Get all documents by envelopeid",
+        "description": "Get all documents related to a given envelope",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Document"
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      },
+      "post": {
+        "operationId": "PostEnvelopesEnvelopeIdDocuments",
+        "summary": "Add document to envelope",
+        "description": "Add a new document to an existing envelope",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/DocumentInput"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
The Body is again empty and the important information will be returned in the Location header which is displayed in the dynamic values list
https://verified-dev.api-docs.io/v2.1/documents/post-envelopes-document-id-documents